gobblin-metrics/build.gradle

```diff
@@ -85,7 +85,7 @@ artifacts {
 
 test {
     useTestNG () {
-      excludeGroups 'ignore'
+      excludeGroups 'ignore', 'performance'
```

Comment:
No we can leave it in the same project, but just in a different package. My main concern is this the code doesn't contain any actual tests, you don't have any Assertions or things like that; so it doesn't really needed to tagged as TestNg tests (e.g. with the `@Test` annotation). You could just make this regular code under `src/main/java/`, and have a shell script that invokes the main method of some class. However, it doesn't really matter, the current setup works too.